lib/lbt/analyzer/XMLTemplateAnalyzer.js

```diff
@@ -402,7 +402,7 @@ class XMLTemplateAnalyzer {
 					}
 				}
 			}
-		} catch (err) {
+		} catch {
```

Comment:
If you don't provide var for the catch block, in some cases it might raise another error for missing variable.

Maybe we can provide an exclusion pattern like it's described in here: https://eslint.org/docs/latest/rules/no-unused-vars#argsignorepattern
and replace the unused vars with underscore where needed (mainly in catch blocks).

```
"no-unused-vars": [
				"error", {
					argsIgnorePattern: "^_",
					varsIgnorePattern: "^_",
					caughtErrorsIgnorePattern: "^_",
				},
			],
```

Reply:
When applying this ESLint rule in this repo all unused vars (not starting with "_") get reported by ESLint obviously.
Should be extend this rule to not throw this error or refactor the whole project by removing all the unused vars?

Reply:
Ignore my comment above. Catch without var is just fine